src/k8s-engine/wsl.ts

```diff
@@ -169,14 +169,19 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     this.k3sHelper.initialize().catch((err) => {
       console.log('k3sHelper.initialize failed: ', err);
     });
-    this.progressTracker = new ProgressTracker((progress) => {
+    this.progressTracker = new ProgressTracker(this, (progress) => {
       this.progress = progress;
       this.emit('progress');
     });
-    this.mobySocketProxyProcess = new BackgroundProcess(this, 'Win32 socket proxy', async() => {
+    const commandArgs = ['docker-proxy', 'serve'];
+    const backgroundProcessLabel = 'Win32 socket proxy';
+
+    this.#lastCommand = `wsl-helper ${ commandArgs.join(' ') }`;
+    this.#lastCommandComment = 'Starting up #{ backgroundProcessLabel } background process';
```

Comment:
This is not true — the background process only runs when we call `start()` on it; at this point it exists but is inactive (and the command & message will be disconnected by the time we try to start it).

Not that this matters — the background process will not throw an exception, it'll just restart, so this will never be seen.

Reply:
Redid this.